static/templates/confirm_dialog.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="modal modal-bg hide fade" id="confirm_dialog_modal" tabindex="-1" role="dialog" aria-labelledby="confirm_dialog_modal" aria-hidden="true">
+<div class="modal new-style modal-bg hide fade" id="confirm_dialog_modal" tabindex="-1" role="dialog" aria-labelledby="confirm_dialog_modal" aria-hidden="true">
```

Comment:
Why do we need `new-style` added here?  The commit message doesn't comment on this detail.

Reply:
Ah, my bad, I missed writing about this. I pulled one commit out for this (with the reasoning in the commit message), which also includes another small change which will allow us to use `confirm_dialog.js` more extensively. 

`new-style` has many styles for buttons etc. Here's how the modal looks like without this class...

![image](https://user-images.githubusercontent.com/55339528/111866537-30185c00-8994-11eb-8f45-78d8960f9408.png)
This wasn't a problem earlier, because `confirm_dialog` was used only in the settings pane, which already has this class in the main `settings_overlay.hbs` file.